### PR TITLE
First pass on docs

### PR DESCRIPTION
**Pull request by drajen**
Review, squash & merge.

### CODE REVIEW

File: helm/charts/hpe-csi-driver/README.md [hunk 1]
```diff
@@ -1,60 +1,65 @@
-# HPE CSI Volume Driver Helm Chart
+# HPE CSI Driver for Kubernetes Helm chart
+The [HPE CSI Driver for Kubernetes](https://github.com/hpe-storage/csi-driver) leverages HPE storage platforms to provide scalable and persistent storage for stateful applications.
 
-## Introduction
-
-The Container Storage Interface (CSI) Volume Driver for Kubernetes leverages HPE Nimble Storage to provide scalable and persistent storage for stateful applications.
+## Prerequisites
+- Upstream Kubernetes version > 1.10
```
Comment (rgcostea) [inline helm/charts/hpe-csi-driver/README.md]: This should be > 1.12

File: helm/charts/hpe-csi-driver/README.md [hunk 2]
```diff
@@ -1,60 +1,65 @@
...
 
-## Features list
+Depending on which [Container Storage Provider](https://github.com/hpe-storage/container-storage-provider) (CSP) is being used, other prerequisites and requirements may apply.
 
-- Persistent Volumes
-- Snapshots with Replication
-- Online Volume Expansion
-- Import Volumes
+### HPE Nimble Storage CSP
+- NimbleOS 3.9 or later
```
Comment (rgcostea) [inline helm/charts/hpe-csi-driver/README.md]: 4.6 or later

File: helm/charts/hpe-csi-driver/README.md [hunk 3]
```diff
@@ -1,60 +1,65 @@
...
-```console
+### Testing the Chart
+To test the chart with the name `hpe-csi`:
+```
 helm test hpe-csi --cleanup
 ```
 
-## Configuration
+### Alternative install method
+In some cases it's more practical provide the local configuration via the `helm` command directly. Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example:
```
Comment (rgcostea) [inline helm/charts/hpe-csi-driver/README.md]: more practical to provide

File: helm/charts/hpe-csi-driver/README.md [hunk 4]
```diff
@@ -1,60 +1,65 @@
...
+```
+helm install --name hpe-csi hpe/hpe-csi-driver \
+--set backend=X.X.X.X --set username=admin --set password=xxxxxxxxx
+```
 
-The following table lists the configurable parameters of the HPE-CSI chart and their default values.
+## Using
+To enable dynamic provisioning of volumes through the use of `PersistentVolumeClaim` API objects, a `StorageClass` needs to be declared on the cluster. Please see the [HPE CSI Driver for Kubernetes](https://github.com/hpe-storage/csi-driver) repository for the official documentation for this Helm chart. Also, it's helpful for be familar with [persistent storage concepts](https://kubernetes.io/docs/concepts/storage/volumes/) in Kubernetes prior to deploying stateful workloads.
```
Comment (rgcostea) [inline helm/charts/hpe-csi-driver/README.md]: helpful to be familiar

File: helm/charts/hpe-flexvolume-driver/README.md [hunk 5]
```diff
@@ -1,61 +1,92 @@
...
 To test the chart with name `hpe-flexvolume`:
-
-```console
+```
 helm test hpe-flexvolume --cleanup
 ```
 
-## Configuration
+## Alternative install method
+In some cases it's more practical provide the local configuration via the `helm` command directly. Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example:
```
Comment (rgcostea) [inline helm/charts/hpe-flexvolume-driver/README.md]: more practical to provide

File: helm/charts/hpe-flexvolume-driver/README.md [hunk 6]
```diff
@@ -1,61 +1,92 @@
...
+```
+helm install --name hpe-flexvolume hpe/hpe-flexvolume-driver \
+--set backend=X.X.X.X --set username=admin --set password=xxxxxxxxx \
+--set protocol=iscsi --set fsType=xfs --set mountConflictDelay=120
+```
 
-The following table lists the configurable parameters of the HPE FlexVolume Driver chart and their default values.
+## Using the HPE FlexVolume Driver for Kubernetes
+To enable dynamic provisioning of `PersistentVolume` through the use of `PersistentVolumeClaim` API objects, a `StorageClass` needs to be declared on the cluster. Please see the [HPE FlexVolume Driver for Kubernetes](https://github.com/hpe-storage/flexvolume-driver) repository for the official documentation for this Helm chart. Also, it's helpful for be familar with [persistent storage concepts](https://kubernetes.io/docs/concepts/storage/volumes/) in Kubernetes prior to deploying stateful workloads.
```
Comment (rgcostea) [inline helm/charts/hpe-flexvolume-driver/README.md]: helpful to be familiar

File: helm/charts/hpe-csi-driver/README.md (hunk 1 above)
Comment (drajen) [reply helm/charts/hpe-csi-driver/README.md]: Fixed

File: helm/charts/hpe-csi-driver/README.md (hunk 2 above)
Comment (drajen) [reply helm/charts/hpe-csi-driver/README.md]: What happened to "all the way to Daytona Beach?" 🏖

File: helm/charts/hpe-csi-driver/README.md (hunk 4 above)
Comment (drajen) [reply helm/charts/hpe-csi-driver/README.md]: Fixed

File: helm/charts/hpe-flexvolume-driver/README.md (hunk 6 above)
Comment (drajen) [reply helm/charts/hpe-flexvolume-driver/README.md]: Fixed

File: helm/charts/hpe-csi-driver/README.md (hunk 2 above)
Comment (rgcostea) [reply helm/charts/hpe-csi-driver/README.md]: I said, "All the way to Emerald Isle."

**Pull request merged**